pkg/activator/net/revision_backends.go

```diff
@@ -227,8 +228,6 @@ func (rw *revisionWatcher) checkDests(dests sets.String) {
 
 		rw.logger.Debug("ClusterIP is no longer healthy.")
 
-		// Send update that we are now inactive (both params invalid).
-		rw.sendUpdate("", nil)
```

Comment:
Why is this update no longer needed?

Reply:
Yes, it should be called for sure. I updated it. Thank you.